heka/usr/share/heka/lua_filters/nginx_redshift.lua

```diff
@@ -69,8 +69,9 @@ function process_message()
         return 0
     end
 
-    local version = string.sub(fields[2], 1, 2)
-    if version ~= "v6" and version ~= "v7" and version ~= "v8" then
+    -- this coercion will need to change when if / when version becomes > 9
+    local version = tonumber(string.sub(fields[2], 2, 2))
```

Comment:
It seems given how close we are to v10 that we should fix the coercion to support v > 9 with something like `tonumber(version:match('v([0-9]+)'))`.